### PR TITLE
feat: NVSHAS-9490 support controller SLSA L3

### PR DESCRIPTION
**Pull request by holyspectral**
1. Provide cross platform/standalone Dockerfile for controller and enforcer.
2. Release.yml to publish SLSA-capable artifacts
3. Make sure third party checksum is checked.
4. Provide build target, test-image, build-image and push-image to sync with rancher.
5. Switch to golang:1.22 as its base image

### CODE REVIEW

File: package/build_controller.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -e
+
+STAGE_DIR=stage
+
+machine=$(uname -m)
+echo "Machine hardware architecture is \"$machine\""
+
+if [ "$machine" == "x86_64" ]; then
```
Comment (pohanhuangtw) [inline package/build_controller.sh:9]: NV only run unit-test in x86_64 env? how about arm?

Comment (holyspectral) [reply package/build_controller.sh:9]: In SLSA we use emulated ARM64 build, in which unit test is very slow, so we only run unit test on amd64.  We will revisit once GitHub's arm64 node becomes generally available.

**Review by pohanhuangtw — APPROVED** [review]
LGTM

File: package/build_controller.sh (hunk 1 above)
Comment (pjbgf) [reply package/build_controller.sh:9]: We should decouple the testing from the building script - that way it is not cross-emulated.
A new workflow for tests can be created, which simply run tests at both `push`/`pull_request` events.

**Review by pjbgf — APPROVED** [review]

**Review by BinX-Suse — APPROVED** [review]

**Pull request merged**